dist/fulcrum-schema.min.js

```diff
@@ -12,7 +12,7 @@ function _defineProperties(target, props) { for (var i = 0; i < props.length; i+
 
 function _createClass(Constructor, protoProps, staticProps) { if (protoProps) _defineProperties(Constructor.prototype, protoProps); if (staticProps) _defineProperties(Constructor, staticProps); Object.defineProperty(Constructor, "prototype", { writable: false }); return Constructor; }
 
-var DATA_ELEMENTS = ['TextField', 'ChoiceField', 'ClassificationField', 'YesNoField', 'PhotoField', 'VideoField', 'AudioField', 'SignatureField', 'BarcodeField', 'DateTimeField', 'TimeField', 'Repeatable', 'AddressField', 'HyperlinkField', 'CalculatedField', 'RecordLinkField', 'AttachmentField', 'CheckboxField', 'DynamicField'];
+var DATA_ELEMENTS = ['TextField', 'ChoiceField', 'ClassificationField', 'YesNoField', 'PhotoField', 'VideoField', 'AudioField', 'SignatureField', 'BarcodeField', 'DateTimeField', 'TimeField', 'Repeatable', 'AddressField', 'HyperlinkField', 'CalculatedField', 'RecordLinkField', 'AttachmentField', 'CheckboxField', 'DynamicField', 'DirectionsField'];
```

Comment:
This is more of a naming thing, but should we try to go more generic with this field? Like Place Or Location?

Reply:
I was thinking that as well. The data is describing a location. In tasks, we are using the location to provide directions. So, the actual type should be LocationField. And then the field name in the Task schema could be called Directions.

Reply:
I agree, I think we could call it _place_ to avoid confusion with the _location_ of the metadata

Reply:
done!